src/statmodels.jl

```diff
@@ -185,18 +218,35 @@ function format_pvc(pv::Number)
     end
 end
 
+function show(io::IO, pv::PValue)
```

Comment:
Can you get rid of `format_pvc` now that you added this function? Code duplication isn't great.